clients/src/main/java/org/apache/kafka/common/metrics/JmxReporter.java

```diff
@@ -72,7 +73,9 @@ public JmxReporter() {
     /**
      * Create a JMX reporter that prefixes all metrics with the given string.
      */
+    @Deprecated
```

Comment:
we should indicate in the javadoc what to replace the deprecated method with, in order to pass in the prefix

Reply:
Added to javadoc